pkg/controllers/cspc-controller/scale.go

```diff
@@ -67,11 +67,9 @@ func (pc *PoolConfig) CreateCSPI(cspc *cstor.CStorPoolCluster) error {
 	if err != nil {
 		return err
 	}
-	err = pc.CreateCSPIDeployment(cspc, cspi)
-	if err != nil {
-		return err
-	}
-	return nil
+      return pc.CreateCSPIDeployment(cspc, cspi)
+
+	return err
```

Comment:
We can remove this line since we are returning from line num 70 this part is unreachable
```suggestion
```